README.md

```diff
@@ -23,6 +23,13 @@ Then you will create an instance of imported class, by choosing one of the follo
     ```
     var factory = CommonFactory.createFromKubernetes(namespace, boxName);
     ```
+    It also can be called by using `createFromArguments(args)` with arguments `--namespace` and `--boxName`.
+1. Create factory with a namespace in Kubernetes, the name of the target th2 box from Kubernetes and the name of context to choose the context from Kube config: 
+    ```
+    var factory = CommonFactory.createFromKubernetes(namespace, boxName, contextName);
+    ```
+    It also can be called by using `createFromArguments(args)` with arguments `--namespace`, `--boxName` and `--contextName`. 
+    ContextName parameter is `@Nullable`; if null, current context will not be changed.
```

Comment:
ContextName parameter is `@Nullable`; if **it is set to null, the** current context will not be changed.